app/views/errors/deleted_pid.html.erb

```diff
@@ -15,7 +15,8 @@ Unless required by applicable law or agreed to in writing, software distributed
 %>
 <div id="main_content_container" class="container-fluid">
   <div class="content" class="col-md-8">
-    <h2>Item(s) being deleted</h2>
-    <p>The item(s) you requested are being deleted from the system. If this is not expected, contact your support staff.</p>
+    <% item = request.original_url.include?("admin/collections") ? "collection" : "item" %>
```

Comment:
Unfortunately, I don't think this is a good test.  The tombstone doesn't have any indication what type of object it was and one can manually enter a url to get this error message for either collection or item with the same tombstoned id.

Reply:
I see what you mean. Since the tombstone isn't limited to its old path and has no indicators on what it used to be, it might be best to remove the test and just do "Item Deleted" for all cases.